.rubocop-https---shopify-github-io-ruby-style-guide-rubocop-yml

```diff
@@ -892,7 +892,7 @@ Lint/FormatParameterMismatch:
   Enabled: true
 
 Lint/HandleExceptions:
-  Enabled: true
+  AllowComments: true
```

Comment:
I don't think it's a good idea to edit this file since it is auto-generated using upstream data.
If this change is absolutely necessary for this PR to pass, the better alternative would be to redefine the setting in the local `.rubocop.yml`

Reply:
I see.. Then you (@shopmike) should push this change to `master` directly then. It may not be ideal but a lot better than pulling the change out from multiple PRs..

Shopify could also consider shipping the Official Style Guide as a Ruby gem so that its updates do not explicitly show up in downstream repos.

Reply:
Yeah, the plan is to push this through as its own PR, just hasn't been a priority.

The config rarely changes and when it does I don't mind the fact it forces everyone to follow it as generally it would only be done when enough people have agreed that it applies to all projects.